expan/core/binning.py

```diff
@@ -132,12 +132,14 @@ def __init__(self, categories):
         Constructor for representation of a categorical bin.
         :param categories: list of categorical values that belong to this bin
         '''
-        if type(categories) is not list:
-            raise ValueError("args for categorical bin is a list of categorical values.")
+        if type(categories) is list:
```

Comment:
what happens if it isn't list?

we could try just to cast to set and catch an exception

Reply:
It is in the next lines. If it's neither list nor set, we will raise an error.

Reply:
Yeah, I'm just saying that
```
categories = set(categories)
```
would do the same job as four lines lines (135 through 138). Please correct me if I'm wrong.

Reply:
Right..Done!